### PR TITLE
Fix typo in log message

### PR DESCRIPTION
**Pull request by adrianmoisey**
**Description of your changes:**
Fixes a typo on a log message

**Which issue is resolved by this Pull Request:**
Resolves no issue

### CODE REVIEW

File: pkg/controller/scyllaoperatorconfig/sync.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ func (opc *Controller) sync(ctx context.Context) error {
 	soc, err := opc.scyllaOperatorConfigLister.Get(naming.SingletonName)
 	if err != nil {
 		if !apierrors.IsNotFound(err) {
-			return fmt.Errorf("can't get sdcyllaoperatorconfig %q: %w", naming.SingletonName, err)
+			return fmt.Errorf("can't get scyllaoperatorconfig %q: %w", naming.SingletonName, err)
```
Comment (mykaul) [inline pkg/controller/scyllaoperatorconfig/sync.go:17]:
```suggestion
			return fmt.Errorf("can't get ScyllaOperatorConfig %q: %w", naming.SingletonName, err)
```

**Comment by tnozicka** [comment]
/ok-to-test

/hold
please clean the commits

**Comment by adrianmoisey** [comment]
> please clean the commits

Does this mean squash into a single commit?

**Comment by tnozicka** [comment]
> Does this mean squash into a single commit?

yep, one line fix doesn't need more then one commit

**Comment by adrianmoisey** [comment]
Done

**Review by tnozicka — APPROVED** [review]
/approve
/lgtm
thanks

**Comment by zimnx** [comment]
commits are squashed
/hold cancel

**Pull request merged**